events/Shire Bree Union.txt

```diff
@@ -0,0 +1,54 @@
+# Remove this line to have the validator check for strings in localisation
+# Audax Validator "!" Ignore_1004
+
+# Remove this line to have the validator check this file
+# Audax Validator "!" Ignore_NEXT

```

Comment:
Unless the validator is throwing some error that we can't ignore (and can't fix), we don't want these lines